.github/.trivyignore

```diff
@@ -0,0 +1 @@
+CVE-2025-48924
```

Comment:
Add a comment why to ignore this CVE (see https://trivy.dev/dev/docs/configuration/filtering/#trivyignore)